### PR TITLE
Add custom theme for MinioHQ

### PR DESCRIPTION
**Pull request by rushenn**
This PR must be merged after https://github.com/minio/gluegun/pull/89

### CODE REVIEW

**Comment by rushenn** [comment]
@deekoder Please merge https://github.com/minio/gluegun/pull/89 before reviewing this.

File: lib/scss/theme.scss [hunk 1]
```diff
@@ -0,0 +1,36 @@
+// Theme Colors
+$text-muted-color: #606060;
+$border-color: #f0f0f0;
+
+
+// Font
+$font-size-base: 18px;
+$font-family: 'Source Sans Pro', sans-serif;
```
Comment (kaankabalak) [inline lib/scss/theme.scss:8]: Don't we also need to load `Source Sans Pro` in the HTML file? Currently I am seeing the font as `sans-serif`. @deekoder told me that we would have to link the font on the index.erb file of minio/gluegun similar to [this](https://github.com/miniohq/gluegun/pull/7/files#diff-55eeb0b31f5e803387ff502806629027R17) and control it with a variable.

**Comment by kaankabalak** [comment]
Thanks for the PR @rushenn, as per @deekoder's feedback, there are just some font related changes we have to make on minio/gluegun and control them with variables:
- We need to load the Source Sans Pro font
- The link headers on the sidebar have to be all uppercase and bold
- The headers have to be bold

**Review by kaankabalak — DISMISSED** [review]
Thanks for the PR @rushenn, as per @deekoder's feedback, there are just some font related changes we have to make on minio/gluegun and control them with variables:
- We need to load the Source Sans Pro font
- The link headers on the sidebar have to be all uppercase and bold
- The headers have to be bold

I'll send a PR to minio/gluegun to fix the issue and then we can take this on top of it with some edits based on the contents of minio/gluegun

**Comment by kaankabalak** [comment]
I sent minio/gluegun#90 to control font related changes using variables. Then, we can apply the changes in #7 to fix the issues listed above.

**Review by kaankabalak — APPROVED** [review]

**Pull request merged**